.github/workflows/integration-tests.yaml

```diff
@@ -74,6 +74,7 @@ jobs:
       - name: Create k8s Kind Cluster
         uses: helm/kind-action@v1.10.0
         with:
+          version: v0.25.0
```

Comment:
It seems that this version specification brought the CI errors. Do we need this?

Reply:
Actually the error is also present with the current version. I've speculatively upgraded it, but it turns out it's the kind-action that stills downloads the `kubectl` binary from `storage.googleapis.com` while the latest versions are now hosted on `dl.k8s.io`.

I've changed it to reference helm/kind-action#127 by SHA until a new version of the action gets released.

Reply:
Oh, I see. Thanks.
In that case, could you open the issue so that we can use the released dedicated tag once the new version is released?

Reply:
Yes, I had already created helm/kind-action#128 earlier :)

Reply:
Thank you for open that!